internal/resource/config/oauth/oauth_auth_server_settings_scopes_exclusive_scope_data_source.go

```diff
@@ -0,0 +1,113 @@
+package oauth
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	client "github.com/pingidentity/pingfederate-go-client/v1125/configurationapi"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/datasource/common/id"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/resource/config"
+	internaltypes "github.com/pingidentity/terraform-provider-pingfederate/internal/types"
+)
+
+// Ensure the implementation satisfies the expected interfaces.
+var (
+	_ datasource.DataSource              = &oauthAuthServerSettingsScopesExclusiveScopesDataSource{}
+	_ datasource.DataSourceWithConfigure = &oauthAuthServerSettingsScopesExclusiveScopesDataSource{}
+)
+
+// Create a Administrative Account data source
+func NewOauthAuthServerSettingsScopesExclusiveScopesDataSource() datasource.DataSource {
```

Comment:
Remove the plurals from this file, since this only gets information about a single scope - "ExclusiveScope" instead of "ExclusiveScopes"